src/Microsoft.AspNetCore.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -610,6 +674,9 @@ private static class NativeDarwinMonoMethods
             public static extern void uv_unref(UvHandle handle);
 
             [DllImport("__Internal", CallingConvention = CallingConvention.Cdecl)]
+            public static extern int uv_fileno(UvHandle handle, ref IntPtr socket);
```

Comment:
I mean, I don't know what the `uv_fileno` function should resolve to on Darwin Mono if taken out. 

With this in it resolves to the libuv function - if I take it out should it resolve to a function shim with body: `throw new NotImplementedExecption("Libuv function not mapped on on Darwin Mono")`?